backend/services/implementations/testSessionService.ts

```diff
@@ -85,6 +85,33 @@ class TestSessionService implements ITestSessionService {
 
     return testSessionDtos;
   }
+
+  async getTestSessionsBySchoolId(
+    schoolId: string,
+  ): Promise<TestSessionResponseDTO[]> {
+    let testSessionDtos: Array<TestSessionResponseDTO> = [];
+
+    try {
+      const testSessions: TestSession[] = await MgTestSession.find({
+        school: { $eq: schoolId },
+      });
+
+      if (!testSessions.length) {
```

Comment:
re: @mahadzaryab1's feedback on my pr -- i think it makes more sense to just return an empty array rather than an error. see his [pr](https://github.com/uwblueprint/jump-math/pull/31/files) as an example